.che-dev.yaml

```diff
@@ -0,0 +1,32 @@
+---
+kind: List
+items:
+ - apiVersion: v1
+   kind: Pod
+   metadata:
+     name: ws
+   spec:
+     containers:
+     - name: dev
+       image: 'eclipse/che-dev:nightly'
+       resources:
+         limits:
+           memory: 1536Mi
+       volumeMounts:
+       -
+         mountPath: /projects
+         name: projects
+     volumes:
+     - name: projects
+       persistentVolumeClaim:
+         claimName: projects
+ - apiVersion: v1
+   kind: PersistentVolumeClaim
+   metadata:
+     name: projects
+   spec:
+     accessModes:
+      - ReadWriteOnce
+     resources:
+       requests:
+         storage: 1Gi
```

Comment:
Since binaries are stored in the same folder where sources are stored - I'm not sure whether 1Gi is enough for Che project folder.
My local Che project folder requires 1.8GiB 
![screenshot_20190130_151727](https://user-images.githubusercontent.com/5887312/51983762-3b515a80-24a2-11e9-8dc6-27d63342b5f4.png)

Also, are there plans to use PVC for maven repo not to download maven dependencies after each workspace restart?